terraform/environments/core-network-services/locals.tf

```diff
@@ -58,7 +58,11 @@ locals {
     "10.102.0.0/16",
     "10.47.0.0/26",
     "10.47.0.64/26",
-    "10.47.0.128/26"
+    "10.47.0.128/26",
+    "10.40.129.79/32"
+  ]
+  noms_dev_vpn_static_routes = [
```

Comment:
LGTM - you'll need to route this through both the `NOMS-Transit-Live-VPN-VNG_1` and `NOMS-Transit-Live-VPN-VNG_2` VPNs.